peer/brontide.go

```diff
@@ -652,7 +702,14 @@ func (p *Brontide) Start() error {
 
 		return lastSerializedBlockHeader[:]
 	}
-	go p.pingHandler(newPingPayload)
+	randPongSize := func() uint16 {
+		return uint16(
+			// We don't need cryptographic randomness here.
+			/* #nosec */
+			rand.Intn(65532),
```

Comment:
So we'll maximally end up here with ~1 kB/s

Reply:
The goal with this was to reach maximum variance to minimize accidental collisions of pings. It's possibly overkill and I can shrink it down but I decided to cover the entire valid input space.